scss/components/_tags.scss

```diff
@@ -18,6 +18,24 @@
   width: 100%;
 }
 
+.tag {
+  background-color: $inverse;
+  border: 1px solid $gray;
+  border-radius: 4px;
+  color: $primary;
+  display: inline-block;
+  font-family: $sans-serif;
+  letter-spacing: -.3px;
+  padding: u(0 .5rem 0 .75rem);
+  margin: u(.5rem 3rem 0 0);
+}
+
+.tag--secondary {
```

Comment:
Nope. The convention is `.base--modifier` and `.base__element` . So if it's just a variant, as in this case, of the same type of element, then it's double dash.